plugins/chat/plugin.rb

```diff
@@ -303,32 +304,29 @@ module ::Chat
   end
 
   register_presence_channel_prefix("chat") do |channel_name|
-    next nil unless channel_name == "/chat/online"
-    config = PresenceChannel::Config.new
-    config.allowed_group_ids = Chat.allowed_group_ids
-    config
+    next if channel_name != "/chat/online"
```

Comment:
This isn't part of the fix, but I figured the small refactoring was worth it to match the other functions in this class.